src/Turbo/src/Twig/TurboStreamListenRendererInterface.php

```diff
@@ -21,7 +21,7 @@
 interface TurboStreamListenRendererInterface
 {
     /**
-     * @param string|object $topic
+     * @param string|object|array<string> $topic
```

Comment:
Purely per curiosity, would this not be a BC break? As (purely) hypothetical implementation would need to adapt ? Or do we ignore the docblock here?

Reply:
I think widening is not a BC break?

Reply:
An alternative way could be add a new method with `@method` phpdoc, which would accept arrays only

Reply:
Arf it seems to be one https://symfony.com/doc/current/contributing/code/bc.html#changing-interfaces

Reply:
Twig function is expendable (no interface), so we could pass an array of topics here

```php
    /**
     * @param object|string $topic
     */
    public function turboStreamListen(Environment $env, $topic, ?string $transport = null): string
    {
```

Reply:
ArrayObject would be too wide I think as it can accept anything, TopicSet/TopicCollection works for me

Reply:
hm, i've basically done with TopicSet and additionaly wrote some tests, but then - how do I create TopicSet in twig? :thinking:

Reply:
That was my first idea ... but then we could even make TopicSet internal i guess ? What do you think?

If all this feels too much, we can also "go back" on your new interface/method idea, i really don't mind .. and don't want to make you lose time :)

Reply:
made it internal :)